pom.xml

```diff
@@ -94,13 +94,14 @@
         <jetty.version>9.4.19.v20190610</jetty.version>
         <jax.rs.api.version>2.1</jax.rs.api.version>
         <jersey.version>2.27</jersey.version>
-        <jackson.version>2.9.8</jackson.version>
-        <spring.boot.version>2.1.3.RELEASE</spring.boot.version>
-        <spring.security.version>5.1.3.RELEASE</spring.security.version>
-        <flyway.version>5.2.1</flyway.version>
+        <jackson.version>2.9.9</jackson.version>
+        <spring.boot.version>2.1.6.RELEASE</spring.boot.version>
+        <spring.security.version>5.1.5.RELEASE</spring.security.version>
+        <flyway.version>5.2.4</flyway.version>
```

Comment:
Do any of these dep version changes require NOTICE file changes? not sure if we mention version numbers there or not.

Reply:
I will double check, but they only would require changes if they brought in need dependencies that also required mentions in the NOTICE, or I guess if the copyright years changed.